### ISSUE TITLE
Req for comparison chart

### ISSUE THREAD

**Issue by jkfnc** [rubra-ai/rubra#123]
Would be good to have a comparison chart like one on this page https://github.com/MeetKai/functionary

**Comment by tybalex** [comment]
hi @jkfnc are you referring this [difference chart](https://github.com/MeetKai/functionary?tab=readme-ov-file#the-differences-between-related-projects)?

**Comment by jkfnc** [comment]
Yes

**Comment by sanjay920** [comment]
we'll consider adding a similar chart in the future.
fwiw, rubra models are trained on all of those classes of function calls plus additional "chained" function calls where subsequent function calls depend on inputs/outputs to prior function calls - something we've seen only proprietary models capable of before rubra models

**Comment by jkfnc** [comment]
thanks, seeing the sample cookbook works great, are there plans to merge tools.cpp to llama.cpp mainstream ? if so , will nodejs requirement be removed ?

**Comment by tybalex** [comment]
> thanks, seeing the sample cookbook works great, are there plans to merge tools.cpp to llama.cpp mainstream ? if so , will nodejs requirement be removed ?

Yes, we plan to merge tools.cpp back to llama.cpp in the future. If that happened, the nodeJS requirement will likely be removed.
